src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -8,7 +8,7 @@ slug: /guides/hardware/servo-calibration
 
 Servos are electromechanical components that allow for the precise control and positioning of the various limbs attached to the NUgus robot. Usually, when a servo is found to not be performing correctly (i.e. seizing or a crunching sound can be heard), it should be removed and replaced with a new servo.
 
-Before a servo is first used, its default parameters need to be updated to ensure correct operation. This is done using Dynamixel Wizard, which can be installed for [Windows, Linux or Mac from the Robotis website](https://emanual.robotis.com/docs/en/software/dynamixel/dynamixel_wizard2/).
+Before a servo is first used, its default parameters need to be updated to ensure correct operation. This is done using Dynamixel Wizard, which can be installed for [Windows, Linux or macOS from the Robotis website](https://emanual.robotis.com/docs/en/software/dynamixel/dynamixel_wizard2/).
```

Comment:
Does anyone know off the top of their head what to update line 31 below to? I assume we want the 2.0 version now and for the knees what does it come up as, X-Series or XH540-W270?